protocols/295c73/fields.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "type": "int",
+    "label": "Sample Count",
+    "name": "sample_count",
+    "default": 24
+  },
+  {
+    "type": "dropDown",
+    "label": "Hold PCR plate on ice until block reaches 98 degrees?",
+    "name": "hold_pcr_plate_on_ice_until_block_reaches_98",
+    "options": [
+      { "label": "YES", "value": "1" },
```

Comment:
we can use the JSON values for booleans here, and not have to typecast them in the Python script. So, we can do:
```
{ "label": "YES", "value": true },
{ "label": "NO", "value": false }
```